crocket/src/main/java/com/crocket/Player.java

```diff
@@ -0,0 +1,71 @@
+package com.crocket;
+
+import java.util.Queue;
+import java.util.LinkedList;
+
+
+public class Player implements IEventListener{
+    int strokes;
+    boolean finished;
+    Queue<Entity> unpassedHoops; //Queue of hoops that have not been passed through yet
+    Ball ball;
+    
+    Player(Ball ball) {
+        this.unpassedHoops =  new LinkedList<Entity>(); 
```

Comment:
Little bit weird naming convention for unpassedHoops if it takes both hoops and peg

Reply:
It has now been renamed to unpassedObstacles